sqlite3/lib/src/wasm/file_system.dart

```diff
@@ -4,11 +4,13 @@ import 'dart:indexed_db';
 import 'dart:math';
 import 'dart:typed_data';
 
-import 'package:path/path.dart' as p show url;
+import 'package:path/path.dart' as p;
```

Comment:
Why did this have to change? I think we should avoid accidentally using anything but the `url` path context.

Reply:
In deed. I use other methods from it. Was simpler to remove it while develop. I'll fix